### DISCUSSION TITLE
Nuxt 3 Vite Glob Imports

### DISCUSSION THREAD

**Discussion by chmking** [nuxt/nuxt#18218] [Questions]
https://vitejs.dev/guide/features.html#glob-import

Glob imports don't seem to work out of the box for Nuxt 3. Is there an alternative method?

**Comment by pi0** [comment]
Hi @chmking. It is a current issue (#1693). Until then you need to use dynamic imports with predictable strings or switch to webpack 5 using `vite: false`
